src/app/components/app/app.styles.css

```diff
@@ -6,3 +6,10 @@ ul#menu li {
   display:inline;
   padding: 10px;
 }
+
+.title{
+  font-size: 3vw;
```

Comment:
Not sure about vw, 'rem' / 'px' is standard

Reply:
view width that means font-size directly proportional to the width of the screen